contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -1698,3 +1698,119 @@ create_db_roles_during_upgrade(PG_FUNCTION_ARGS)
 	PG_RETURN_INT32(0);
 }
 
+static void
+exec_revoke_create_privilege_from_guest_user_db(List *stmt_list, char *physical_schema, char *guest_role)
+{
+	Node *stmts;
+	Oid save_userid;
+	int save_sec_context;
+	int i = 0;
+	ListCell	*parsetree_item;
+
+	GetUserIdAndSecContext(&save_userid, &save_sec_context);
+	stmts = parsetree_nth_stmt(stmt_list, i++);
+	update_GrantStmt(stmts, quote_identifier(physical_schema), NULL, quote_identifier(guest_role), NULL);
```

Comment:
1. We can remove quote_identifier here. 
2. `stmts = parsetree_nth_stmt(stmt_list, i++);` could be `stmt = parsetree_nth_stmt(stmt_list, 0);`. There is only one statement, lets remove `i`.